openedx/features/caliper_tracking/tests/expected/problem_check_browser.json

```diff
@@ -0,0 +1,39 @@
+{
+  "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+  "action": "Submitted",
+  "actor": {
+    "id": "http://edx.devstack.lms:18000/u/ali",
```

Comment:
edx.devstack.lms should be changed to localhost:18000

Reply:
done in the latest commit.